lib/elixir/src/elixir_module.erl

```diff
@@ -226,11 +228,12 @@ build(Line, File, Module, Lexical) ->
 
 eval_form(Line, Module, Data, Block, Vars, E) ->
   {Value, EE} = elixir_compiler:eval_forms(Block, Vars, E),
-  elixir_overridable:store_pending(Module),
+  Pairs1 = elixir_overridable:store_pending(Module),
   EV = elixir_env:linify({Line, reset_env(EE)}),
   EC = eval_callbacks(Line, Data, before_compile, [EV], EV),
-  elixir_overridable:store_pending(Module),
-  {Value, EC}.
+  Pairs2 = elixir_overridable:store_pending(Module),
+  OverridablePairsNotOverridden = Pairs1 ++ Pairs2,
```

Comment:
Suggestions for name: `OverridablePairs = OverridablePre ++ OverridablePos`. But `Pairs1` and `Pairs2` is also fine by me.